minecraft_servers/mush/manifest.json

```diff
@@ -0,0 +1,96 @@
+{
+  "server_name": "mush",
+  "nice_name": "Mush",
+  "direct_ip": "jogar.mush.com.br",
+  "server_wildcards": [
+    "%.mush.com.br",
+    "%.mushmc.com",
+    "%.muu.sh"
+  ],
+  "supported_languages": [
+    "pt"
+  ],
+  "social": {
+    "web": "https://mush.com.br",
+    "web_shop": "https://mush.com.br/loja",
+    "web_support": "https://mush.com.br/r/ajuda",
```

Comment:
Cannot be a forum
Has to be some kind of FAQ or a contact form

Reply:
I've removed it